gateway/src/api/latest.rs

```diff
@@ -103,161 +93,6 @@ impl StatusResponse {
     }
 }
 
-#[instrument(skip(service))]
-#[utoipa::path(
-    get,
-    path = "/projects/{project_name}",
-    responses(
-        (status = 200, description = "Successfully got a specific project information.", body = shuttle_common::models::project::Response),
-        (status = 500, description = "Server internal error.")
-    ),
-    params(
-        ("project_name" = String, Path, description = "The name of the project."),
-    )
-)]
-async fn get_project(
```

Comment:
Wondering if we should keep this one too. Of course, removing any of them can be readded later on, so not sure what is the strategy. We clean everything and re-add them hooked to the proper deps (e.g. deployer/builder)?

Reply:
Yes, I added this back in in the latest commit